<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="QA-SWAMYK" name="com.quoteorder.prospect.Add_NewShipping_Address" tests="6" failures="0" timestamp="2020-12-15T14:14:50 IST" time="333.435" errors="1">
  <testcase name="testOECAccount" time="32.874" classname="com.quoteorder.prospect.Add_NewShipping_Address"/>
  <system-out/>
  <testcase name="testNAVtoCPQ" time="65.058" classname="com.quoteorder.prospect.Add_NewShipping_Address"/>
  <system-out/>
  <testcase name="testTransactionDetails" time="123.626" classname="com.quoteorder.prospect.Add_NewShipping_Address"/>
  <system-out/>
  <testcase name="testCalculateChargesTab" time="38.563" classname="com.quoteorder.prospect.Add_NewShipping_Address">
    <error type="java.lang.ArrayIndexOutOfBoundsException" message="1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 1
at com.softclouds.framework.SeleniumWrapper.getByLocator(SeleniumWrapper.java:203)
at com.softclouds.framework.SeleniumWrapper.getWebElement(SeleniumWrapper.java:304)
at com.softclouds.framework.SeleniumWrapper.compare(SeleniumWrapper.java:138)
at com.quoteorder.prospect.Add_NewShipping_Address.testCalculateChargesTab(Add_NewShipping_Address.java:214)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testCalculateChargesTab -->
  <system-out/>
  <testcase name="testApprovalsTab" time="28.272" classname="com.quoteorder.prospect.Add_NewShipping_Address"/>
  <system-out/>
  <testcase name="testReturnCPQToOEC" time="45.042" classname="com.quoteorder.prospect.Add_NewShipping_Address"/>
  <system-out/>
</testsuite> <!-- com.quoteorder.prospect.Add_NewShipping_Address -->
